saviSimulation/savi/commandStation/SocketConnector.java

```diff
@@ -0,0 +1,100 @@
+package savi.commandStation;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.DatagramSocket;
+import java.net.InetAddress;
+import java.net.SocketException;
+import java.net.UnknownHostException;
+
+public class SocketConnector {
+	
+	private int myPort;
+	private int otherPort;
+	
+	private MessageHandler inboxHandler;
+	
+
+	
+	public SocketConnector(MessageHandler msgHandler, int myPort,
+			int otherPort) {
+		
+		inboxHandler = msgHandler;
+		this.myPort = myPort;
+		this.otherPort = otherPort;
+		
+	}
+
+
+
+	public void messageOut(String msg) {
+		System.out.println("Message OUT: "+msg);
+		//get data for message
+		byte [] data = msg.getBytes();
+         
+        try {
+        	DatagramSocket socket = new DatagramSocket();
+        	socket.setBroadcast(true);
+
+        	InetAddress aHost = InetAddress.getLocalHost();
```

Comment:
This should be updated to allow for a configurable address. Perhaps on a future version.

Reply:
To be addressed by #157 .